bower.json

```diff
@@ -11,11 +11,11 @@
   "license": "Apache License, Version 2.0",
   "repository": {
     "type": "git",
-    "url": "git://github.com/mtscout6/react-router-bootstrap.git"
+    "url": "git://github.com/react-router/react-router-bootstrap.git"
   },
   "dependencies": {
-    "react-router": "^0.11.6",
-    "react-bootstrap": "^0.13.2"
+    "react-bootstrap": ">=0.15",
+    "react-router": ">=0.13.1"
```

Comment:
I stopped using bower for this package, actually, in part because of this issue, so I don't have an opinion on this any more. I just thought I'd try to close this issue since I was submitting the other PR as well.